internal/cluster/clusterworkflow/activity_create_node_pool.go

```diff
@@ -118,6 +118,14 @@ func (a CreateNodePoolActivity) Execute(ctx context.Context, input CreateNodePoo
 			)
 		}
 
+		var clusterTags map[string]string
+		if eksCluster.Cluster.Tags != nil {
+			clusterTags = make(map[string]string, len(eksCluster.Cluster.Tags))
+			for _, tag := range eksCluster.Cluster.Tags {
+				clusterTags[tag.Key] = tag.Value
+			}
+		}
```

Comment:
If this is such a frequent operation (going from `[]ClusterTag` to `map[string]string`), I'd consider using a helper function.

Reply:
Or just use `map[string]string`?